scripts/stack-compose-runtime.sh

```
@@ -1751,6 +1952,22 @@ append_sabnzbd_service_body() {
 
+  local sab_host_binding=""
+  if [[ "$sab_expose_lan" == "1" && "${SABNZBD_USE_VPN}" != "1" ]]; then
+    local resolved_host=""
+    arr_compose_resolve_lan_binding_host resolved_host
+    if [[ -n "$resolved_host" ]]; then
+      sab_host_binding="$resolved_host"
+    fi
+  fi
+
+  if [[ -n "$sab_host_binding" ]]; then
+    local sab_port_placeholder='${SABNZBD_PORT}'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)